.gitignore

```
@@ -73,9 +73,6 @@ build/
 /.idea/libraries
 /captures
 .externalNativeBuild
```

```suggestion
.externalNativeBuild
gradle/
gradlew
gradlew.bat
```
Can we add these back to the gitignore